config/groups.yaml

```
@@ -120,6 +120,12 @@
     - "2666525"
     - "2328859"
   id: 189749
+- name: "academic weapons"
+  members:
+    -"2567033"
+    -"2591194"
+    -"2566752"
+    -"2565473"
```

You need spaces between the dashes and the string values:
```suggestion
    - "2567033"
    - "2591194"
    - "2566752"
    - "2565473"
```